### PR TITLE
[TEAM2-374] Remove Deprecated Testnets

### PR DESCRIPTION
**Pull request by derHowie**
## What changed (plus any additional context for devs)
I removed Rinkeby, Kovan, and Ropsten in this PR. #4009 is a sister PR to this one, they should probably be looked at in tandem. 

## What to test
Make sure that they're gone.

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [x] Did you test both iOS and Android?
- [x] If your changes are visual, did you check both the light and dark themes?
- [x] Added e2e tests? If not, please specify why
- [x] If you added new critical path files, did you update the CODEOWNERS file?
- [x] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

**Review by estebanmino — APPROVED** [review]
changes lgtm, looking good on the app as well, should we wait for to be ready to go and merge it right before merging https://github.com/rainbow-me/rainbow/pull/4001/files# ?

**Review by skylarbarrera — APPROVED** [review]
looks like theyre all gone 😢

**Review by brunobar79 — APPROVED** [review]

**Pull request merged**
